timesketch/lib/analyzers/geoip.py

```diff
@@ -274,8 +280,11 @@ def run(self):
                     ip_addresses[ip_addr][ip_address_field].append(event)
 
         for ip_address, ip_address_fields in ip_addresses.items():
-            with self.GEOIP_CLIENT() as client:  # pylint: disable=E1102
-                response = client.ip2geo(ip_address)
+            try:
+                with self.GEOIP_CLIENT() as client:  # pylint: disable=E1102
+                    response = client.ip2geo(ip_address)
+            except GeoIPClientError as error:
+                return f'GeoIP Client error - {error}'
```

Comment:
This will stop the entire run, is that what you want here? Maybe just log the error, and add it to a `fail` counter and report # fails in the end?